--- bgqe\setup-bgqe.tp2
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\bgqe\\setup-bgqe.tp2"	
@@ -11,7 +11,7 @@ BACKUP ~bgqe/Backup~
 AUTHOR ~Please post in one of the forums as listed in the readme~
 
 //MODDER
-VERSION ~v25.6~ 
+VERSION ~v25.6 BWS-FR Fix~
 
 README ~bgqe/readmes/%LANGUAGE%.bgqe.txt~ ~bgqe/readmes/english.bgqe.txt~
 
@@ -1712,11 +1712,11 @@ END
 
 COPY ~bgqe/farmer/cre/WAGHOUL.CRE~ ~override~
 SAY NAME1 @2002
-SAY NAME1 @2002
+SAY NAME2 @2002
 
 COPY ~bgqe/farmer/cre/WAFARM.CRE~ ~override~
 SAY NAME1 @2003
-SAY NAME1 @2003
+SAY NAME2 @2003
 
 
 ACTION_IF GAME_IS ~bg1 totsc~ BEGIN
