--- bg1npc\phase2\tpa\bg1npc_kagqst.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\bg1npc\\phase2\\tpa\\bg1npc_kagqst.tpa"	
@@ -132,4 +132,9 @@ COMPILE EVALUATE_BUFFER ~bg1npc/phase2/baf/X#KABAND.BAF~
 COMPILE EVALUATE_BUFFER ~bg1npc/phase2/baf/X#BANDK.BAF~
 
 /* Add Journal entries for BGEE */
-ADD_JOURNAL EXISTING TITLE (#31350) @35 @49 @54 USING ~bg1npc/tra/%LANGUAGE%/x#kagqst.tra~
+ACTION_IF GAME_IS ~bgee~ THEN BEGIN
+  ADD_JOURNAL EXISTING TITLE (#31350) @35 @49 @54 USING ~bg1npc/tra/%LANGUAGE%/x#kagqst.tra~
+END
+ACTION_IF GAME_IS ~eet~ THEN BEGIN
+  ADD_JOURNAL EXISTING TITLE (#231350) @35 @49 @54 USING ~bg1npc/tra/%LANGUAGE%/x#kagqst.tra~
+END
