--- TDDz\TDDz.tp2
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\TDDz\\TDDz.tp2"	
@@ -1,7 +1,7 @@
 BACKUP ~TDDz/backup~
 //AUTHOR ~shadowlich~
 SUPPORT ~https://forums.beamdog.com/discussion/4841/the-darkest-day/~
-VERSION ~v1.3.1~
+VERSION ~v1.3.1 BWS-FR Fix~
 
 ALWAYS
 	INCLUDE ~%MOD_FOLDER%/Lib/Helpers.tpa~
@@ -90,6 +90,7 @@ LANGUAGE 	~Spanish(Clan REO)~
 
 
 BEGIN @901000
+	DESIGNATED 0
 	REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet~ ~~
 	COPY ~TDD/Bam~ ~override~
         INCLUDE ~%MOD_FOLDER%/Lib/Importer.tpa~
@@ -109,6 +110,7 @@ END
 COPY ~TDDz/PVRZ~ ~override~
 
 BEGIN @901001
+	DESIGNATED 1
 	REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet~ ~~
 	REQUIRE_COMPONENT "setup-TDDz.tp2" "0" @901011
 
@@ -120,6 +122,7 @@ BEGIN @901001
 COPY ~TDDz/PVRZ~ ~override~
 
 BEGIN @901002
+	DESIGNATED 2
 	REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet~ ~~
 	REQUIRE_COMPONENT "setup-TDDz.tp2" "0" @901011
 
@@ -131,6 +134,7 @@ BEGIN @901002
 COPY ~TDDz/PVRZ~ ~override~
 
 BEGIN @901003
+	DESIGNATED 3
 	REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet~ ~~
 	REQUIRE_COMPONENT "setup-TDDz.tp2" "0" @901011
 
@@ -140,6 +144,7 @@ BEGIN @901003
 		LAF DeleteTempFolder STR_VAR aComp = ~PURSKAL~ END
 
 BEGIN @901004
+	DESIGNATED 4
 	REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet~ ~~
 	REQUIRE_COMPONENT "setup-TDDz.tp2" "0" @901011
 
@@ -151,6 +156,7 @@ BEGIN @901004
 COPY ~TDDz/PVRZ~ ~override~
 
 BEGIN @901005
+	DESIGNATED 5
 	REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet~ ~~
 	REQUIRE_COMPONENT "setup-TDDz.tp2" "0" @901011
 
@@ -162,6 +168,7 @@ BEGIN @901005
 COPY ~TDDz/PVRZ~ ~override~
 
 BEGIN @901010
+	DESIGNATED 6
 	REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet~ ~~
 	REQUIRE_COMPONENT "setup-TDDz.tp2" "0" @901011
 
@@ -171,6 +178,61 @@ BEGIN @901010
 		LAF DeleteTempFolder STR_VAR aComp = ~ROBILARD~ END
 
 
+/////////////////////////////
+BEGIN ~Use BGGO extended BG1 areas for TDDz~
+//REQUIRE_PREDICATE (MOD_IS_INSTALLED "TDDZ/SETUP-TDDZ.TP2" "2") OR (MOD_IS_INSTALLED "TDDZ/SETUP-TDDZ.TP2" "3") OR (MOD_IS_INSTALLED "TDDZ/SETUP-TDDZ.TP2" "4") "No TDD component detected."
+//REQUIRE_COMPONENT "BGGO/SETUP-BGGO.TP2" "0" "The main component must be installed before"
+REQUIRE_PREDICATE MOD_IS_INSTALLED "bggo.tp2" (ID_OF_LABEL "bggo.tp2" "baldurs_gate_graphical_overhaul" ) "No BGGO is installed"
+REQUIRE_PREDICATE (MOD_IS_INSTALLED "TDDZ/SETUP-TDDZ.TP2" "2") OR (MOD_IS_INSTALLED "TDDZ/SETUP-TDDZ.TP2" "3") OR (MOD_IS_INSTALLED "TDDZ/SETUP-TDDZ.TP2" "4") "No TDD component detected."
+REQUIRE_PREDICATE GAME_IS ~eet bgt~ ~Only for bgt or eet~
+
+ACTION_IF GAME_IS ~eet~ BEGIN
+	ACTION_DEFINE_ASSOCIATIVE_ARRAY remapped_tis_tdd BEGIN
+		"DD1900"    => "BG1900" //%BanditCamp%"
+		"DD2300"    => "BG2300" //%FriendlyArmInn%"
+		"DD3300"    => "BG3300" //%Beregost%"
+		"DD8000"    => "BG2900" //%Larswood%"
+	END
+END
+
+ACTION_IF GAME_IS ~bgt~ BEGIN
+	ACTION_DEFINE_ASSOCIATIVE_ARRAY remapped_tis_tdd BEGIN
+		"DD1900"    => "AR8700" //%BanditCamp%"
+		"DD2300"    => "AR6800" //%FriendlyArmInn%"
+		"DD3300"    => "AR6700" //%Beregost%"
+		"DD8000"    => "AR9000" //%Larswood%"
+	END
+END
+
+ACTION_DEFINE_ASSOCIATIVE_ARRAY turn_off_at_night BEGIN // turn off sleeping animals at night
+		bird => 1
+		eagle => 1
+		vultur => 1
+		seagul => 1
+END
+
+ACTION_PHP_EACH remapped_tis_tdd AS TDDARE => BG1ARE BEGIN
+	ACTION_IF FILE_EXISTS_IN_GAME ~%TDDARE%.are~ BEGIN
+		COPY_EXISTING ~%TDDARE%.are~ ~override~
+		PATCH_IF SOURCE_SIZE > 0x11b BEGIN
+			PATCH_IF (((BYTE_AT 0x48) << 7) & BIT7) BEGIN // Toggle only outdoor areas
+				WRITE_BYTE 0x48 (THIS | BIT6 | BIT1 | BIT2) // Toggle extended night flag (BIT6) and day/night flag (BIT1) and weather flag (BIT2)
+				GET_OFFSET_ARRAY actors 0x54 4 0x58 2 0 0 0x110		// turn off creatures at night time
+				PHP_EACH actors AS i => off BEGIN
+					READ_ASCII off + 0x80 cre_resref
+					TO_LOWER cre_resref
+					PATCH_IF VARIABLE_IS_SET $turn_off_at_night("%cre_resref%") BEGIN
+						WRITE_LONG off + 0x40 0x1ffe0 // 05:30 to 17:29
+					END
+				END
+			END
+			WRITE_ASCIIE 0x8 ~%BG1ARE%~ #8
+		END
+		BUT_ONLY
+	END
+END
+
+
 //=============================================================================
 /* USED DURING INITIAL DEVELOPMENT - NOT USED NOW
 //BEGIN ~DEBUG~
@@ -178,4 +240,4 @@ BEGIN @901010
 //	DESIGNATED 99
 //	INCLUDE ~%MOD_FOLDER%/Lib/Translations.tpa~
 //		LAM DispatchTraFiles
-*/
+*/
\ No newline at end of file
