--- spell_rev\lib\kreso_eestatsr.tph
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\spell_rev\\lib\\kreso_eestatsr.tph"	
@@ -67,7 +67,7 @@ END
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -75,12 +75,12 @@ END
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~bdsha12a.spl~	override		// ??
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -88,12 +88,12 @@ END
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~bdshld02.spl~	override		// fire attack from shield
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -101,12 +101,12 @@ END
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~ohbeflam.spl~	override		// flames!
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -114,12 +114,12 @@ END
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~balshld2.spl~	override		// some balthy crp
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -127,12 +127,12 @@ END
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~balth01b.spl~	override		// balthazar fireshield
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -140,12 +140,12 @@ END
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spcl236.spl~	override		// sun soul monk soulray
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -153,12 +153,12 @@ END
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spcl237d.spl~	override		// sun soul monk fireshield
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -171,7 +171,7 @@ COPY_EXISTING	~spcl237d.spl~	override		// sun soul monk fireshield
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -179,12 +179,12 @@ COPY_EXISTING	~spcl237d.spl~	override		// sun soul monk fireshield
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spcl239a.spl~	override		// sun soul monk blinding attack, assuming fire causes blindness
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -192,12 +192,12 @@ COPY_EXISTING	~spcl237d.spl~	override		// sun soul monk fireshield
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spdd03.spl~	override		// dragon disciple breath attack
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -205,12 +205,12 @@ COPY_EXISTING	~spcl237d.spl~	override		// sun soul monk fireshield
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 		COPY_EXISTING	~spctmd01.spl~	override		//fireball
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -218,12 +218,12 @@ COPY_EXISTING	~spcl237d.spl~	override		// sun soul monk fireshield
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spin539.spl~	override		//magma bolt
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -236,7 +236,7 @@ COPY_EXISTING	~spcl237d.spl~	override		// sun soul monk fireshield
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -249,7 +249,7 @@ COPY_EXISTING	~spin560.spl~	override		// Cinder Shower
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -262,7 +262,7 @@ COPY_EXISTING	~spin561.spl~	override		// fire giant lava attack
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -275,7 +275,7 @@ COPY_EXISTING	~spin561.spl~	override		// fire giant lava attack
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -289,7 +289,7 @@ COPY_EXISTING	~spin819.spl~	override		// Lava burst
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -302,7 +302,7 @@ COPY_EXISTING	~spin885.spl~	override		// kamikaze explosion
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -310,13 +310,13 @@ COPY_EXISTING	~spin885.spl~	override		// kamikaze explosion
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
-	
+
+
 	COPY_EXISTING	~spin894.spl~	override		// Red dragon breath
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -329,7 +329,7 @@ COPY_EXISTING	~spin926.spl~	override		// water jet (fire damage, steam or whatev
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -342,7 +342,7 @@ COPY_EXISTING	~spin926.spl~	override		// water jet (fire damage, steam or whatev
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -355,7 +355,7 @@ COPY_EXISTING	~spin938.spl~	override		//  Flame fan
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -368,7 +368,7 @@ COPY_EXISTING	~spin938.spl~	override		//  Flame fan
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -381,7 +381,7 @@ COPY_EXISTING	~spin956.spl~	override		//  Hell hound flame breath
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -394,7 +394,7 @@ COPY_EXISTING	~spin987.spl~	override		//  Beholder scorcher
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -402,12 +402,12 @@ COPY_EXISTING	~spin987.spl~	override		//  Beholder scorcher
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~%sppr116%.spl~	override		//  sunscorch
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -415,12 +415,12 @@ COPY_EXISTING	~spin987.spl~	override		//  Beholder scorcher
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 		COPY_EXISTING	~%sppr216%.spl~	override		//  Fire trap
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -433,7 +433,7 @@ COPY_EXISTING	~sppr503.spl~	override		//  Flamestrike
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -446,7 +446,7 @@ COPY_EXISTING	~sppr503.spl~	override		//  Flamestrike
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -460,7 +460,7 @@ COPY_EXISTING	~sppr730d.spl~	override		//  Flaming death aura
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -473,7 +473,7 @@ COPY_EXISTING	~sppr952d.spl~	override		//  some fireshield
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -486,7 +486,7 @@ COPY_EXISTING	~sppr985.spl~	override		//  Flamestrike trap
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -494,12 +494,12 @@ COPY_EXISTING	~sppr985.spl~	override		//  Flamestrike trap
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spwi001.spl~	override		//  fireball trap
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -513,7 +513,7 @@ COPY_EXISTING	~spwi022.spl~	override		//  muck(!) trap, fire damage
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -526,7 +526,7 @@ COPY_EXISTING	~spwi103.spl~	override		//  burning hands
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -539,7 +539,7 @@ COPY_EXISTING	~spwi217.spl~	override		//  Aganaza scorcher
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -552,7 +552,7 @@ COPY_EXISTING	~spwi217.spl~	override		//  Aganaza scorcher
 //COPY_EXISTING ~spwi303e.spl~ ~override~
 //	LPF	ADD_SPELL_EFFECT INT_VAR
 //		insert_point = 0
-//		opcode = 324
+//		opcode = 318
 //		target = 2
 //		parameter2 = FIRE
 //		duration = 1
@@ -565,7 +565,7 @@ COPY_EXISTING	~spwi304.spl~	override		// fireball
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -578,7 +578,7 @@ COPY_EXISTING	~spwi418d.spl~	override		// fireshield red
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -591,7 +591,7 @@ COPY_EXISTING	~spwi523.spl~	override		// sunfire
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -599,12 +599,12 @@ COPY_EXISTING	~spwi523.spl~	override		// sunfire
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spwi523d.spl~	override		// sunfire
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -617,7 +617,7 @@ COPY_EXISTING	~spwi712.spl~	override		// Delayed blast fireball
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -630,7 +630,7 @@ COPY_EXISTING	~spwi712.spl~	override		// Delayed blast fireball
 //COPY_EXISTING ~spwi810.spl~ ~override~
 //    LPF ADD_SPELL_EFFECT INT_VAR
 //        insert_point = 0
-//        opcode = 324
+//        opcode = 318
 //        target = 2
 //        parameter2 = FIRE
 //        duration = 1
@@ -638,12 +638,12 @@ COPY_EXISTING	~spwi712.spl~	override		// Delayed blast fireball
 //        resource = EVAL ~%SOURCE_RES%~
 //     END
 //BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spwi911.spl~	override		// Meteor swarm
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -656,7 +656,7 @@ COPY_EXISTING	~spwi922.spl~	override		// Dragon's Breath
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -670,7 +670,7 @@ COPY_EXISTING	~spwi922.spl~	override		// Dragon's Breath
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -683,7 +683,7 @@ COPY_EXISTING	~spwi957.spl~	override		// "RED" Fireball
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -696,7 +696,7 @@ COPY_EXISTING	~spwi979.spl~	override		// sarevok flamestrike - check if buggy or
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -709,7 +709,7 @@ COPY_EXISTING	~spwish24.spl~	override		// meteor swarm on caster
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -717,13 +717,13 @@ COPY_EXISTING	~spwish24.spl~	override		// meteor swarm on caster
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
-	
+
+
 COPY_EXISTING	~ohbwi304.spl~	override		// fireball
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = FIRE
 				duration = 1
@@ -739,7 +739,7 @@ COPY_EXISTING	~ohbicew.spl~	override		// wall of ice, does crushing as well but
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -747,12 +747,12 @@ COPY_EXISTING	~ohbicew.spl~	override		// wall of ice, does crushing as well but
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~1tarss.spl~	override		// cold fireshield (???)
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -760,12 +760,12 @@ COPY_EXISTING	~ohbicew.spl~	override		// wall of ice, does crushing as well but
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spcryo01.spl~	override		// cryonax cone of cold
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -773,12 +773,12 @@ COPY_EXISTING	~ohbicew.spl~	override		// wall of ice, does crushing as well but
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spcryod.spl~	override		// cryonax fireshield
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -786,12 +786,12 @@ COPY_EXISTING	~ohbicew.spl~	override		// wall of ice, does crushing as well but
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 COPY_EXISTING	~spin562.spl~	override		// scalding steam
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -800,12 +800,12 @@ COPY_EXISTING	~spin562.spl~	override		// scalding steam
 		END
 	BUT_ONLY IF_EXISTS
 
-	
+
 	COPY_EXISTING	~spin723.spl~	override		// Dragon rain (cold, can slay but assuming slays by ice)
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -813,12 +813,12 @@ COPY_EXISTING	~spin562.spl~	override		// scalding steam
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spin833.spl~	override		// Silver dragon cold attack
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -826,12 +826,12 @@ COPY_EXISTING	~spin562.spl~	override		// scalding steam
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spin936.spl~	override		// Mephit ice shard
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -839,12 +839,12 @@ COPY_EXISTING	~spin562.spl~	override		// scalding steam
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~%sppr418%.spl~	override		// ice storm druid
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -852,12 +852,12 @@ COPY_EXISTING	~spin562.spl~	override		// scalding steam
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 			COPY_EXISTING	~spdr501.spl~	override		// cone of cold avenger
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -867,12 +867,12 @@ COPY_EXISTING	~spin562.spl~	override		// scalding steam
 	BUT_ONLY IF_EXISTS
 
 
-	
+
 	COPY_EXISTING	~sppr951d.spl~	override		// Fireshield
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -880,12 +880,12 @@ COPY_EXISTING	~spin562.spl~	override		// scalding steam
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spwi015.spl~	override		// trap cold
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -893,12 +893,12 @@ COPY_EXISTING	~spin562.spl~	override		// scalding steam
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spwi018.spl~	override		// trap cold
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -906,12 +906,12 @@ COPY_EXISTING	~spin562.spl~	override		// scalding steam
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 		COPY_EXISTING	~%spwi327%.spl~	override		// icelance
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -919,12 +919,12 @@ COPY_EXISTING	~spin562.spl~	override		// scalding steam
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~%sppr326%.spl~	override		// icelance druid
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -937,7 +937,7 @@ COPY_EXISTING	~spwi403d.spl~	override		// cold fireshield
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -950,7 +950,7 @@ COPY_EXISTING	~spwi403d.spl~	override		// cold fireshield
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -958,12 +958,12 @@ COPY_EXISTING	~spwi403d.spl~	override		// cold fireshield
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spwi503.spl~	override		// cone of cold
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -971,12 +971,12 @@ COPY_EXISTING	~spwi403d.spl~	override		// cold fireshield
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~ubsnobr.spl~	override		// snow golem frost ray
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -984,12 +984,12 @@ COPY_EXISTING	~spwi403d.spl~	override		// cold fireshield
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 		COPY_EXISTING	~spwi818.spl~	override		// icy grasp
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -997,12 +997,12 @@ COPY_EXISTING	~spwi403d.spl~	override		// cold fireshield
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 			COPY_EXISTING	~spwi818d.spl~	override		// icy grasp
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = COLD
 				duration = 1
@@ -1019,7 +1019,7 @@ COPY_EXISTING	~spcl722.spl~	override		// Talos lighting bolt
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1032,7 +1032,7 @@ COPY_EXISTING	~spdr301.spl~	override		//  avenger lighting bolt
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1045,7 +1045,7 @@ COPY_EXISTING	~spdr601.spl~	override		//  avenger lighting bolt
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1058,7 +1058,7 @@ COPY_EXISTING	~spin579.spl~	override		// power amp
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1066,12 +1066,12 @@ COPY_EXISTING	~spin579.spl~	override		// power amp
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 COPY_EXISTING	~spin597.spl~	override		// blue dragon lighting attack
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1084,7 +1084,7 @@ COPY_EXISTING	~spin714.spl~	override		//  clestial bolt
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1097,7 +1097,7 @@ COPY_EXISTING	~spin932.spl~	override		//  mephit lighting bolt
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1110,7 +1110,7 @@ COPY_EXISTING	~spin933.spl~	override		//  mephit lighting bolt
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1123,7 +1123,7 @@ COPY_EXISTING	~sppr302.spl~	override		// call lighting
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1136,7 +1136,7 @@ COPY_EXISTING	~sppr304.spl~	override		// glyph of warding
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1149,7 +1149,7 @@ COPY_EXISTING	~sppr987.spl~	override		// call lighting trap
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1162,7 +1162,7 @@ COPY_EXISTING	~spwi002.spl~	override		//  lighting trap
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1175,7 +1175,7 @@ COPY_EXISTING	~spwi017.spl~	override		// minor lighting trap
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1188,7 +1188,7 @@ COPY_EXISTING	~spwi025.spl~	override		// lighting orb
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1201,7 +1201,7 @@ COPY_EXISTING	~spwi026.spl~	override		// lighting orb 2
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1214,7 +1214,7 @@ COPY_EXISTING	~spwi027.spl~	override		// orb 3
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1227,7 +1227,7 @@ COPY_EXISTING	~spwi308.spl~	override		// lighting bolt
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1240,7 +1240,7 @@ COPY_EXISTING	~spwi615.spl~	override		// chain lighting
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1248,12 +1248,12 @@ COPY_EXISTING	~spwi615.spl~	override		// chain lighting
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spwi615a.spl~	override		// chain lighting
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1261,12 +1261,12 @@ COPY_EXISTING	~spwi615.spl~	override		// chain lighting
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spwi615b.spl~	override		// chain lighting
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1274,12 +1274,12 @@ COPY_EXISTING	~spwi615.spl~	override		// chain lighting
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~bdhalb01.spl~	override		// some halberd from sod
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1287,12 +1287,12 @@ COPY_EXISTING	~spwi615.spl~	override		// chain lighting
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~ohbcalll.spl~	override		// ???
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1300,13 +1300,13 @@ COPY_EXISTING	~spwi615.spl~	override		// chain lighting
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
-	
+
+
 	COPY_EXISTING	~ohbwi308.spl~	override		// lighting bolt
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ELEC
 				duration = 1
@@ -1322,7 +1322,7 @@ COPY_EXISTING	~spwi615.spl~	override		// chain lighting
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ACID
 				duration = 1
@@ -1330,12 +1330,12 @@ COPY_EXISTING	~spwi615.spl~	override		// chain lighting
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 		COPY_EXISTING	~ohrgrog.spl~	override		// ??
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ACID
 				duration = 1
@@ -1343,12 +1343,12 @@ COPY_EXISTING	~spwi615.spl~	override		// chain lighting
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~bdmepwat.spl~	override		// ???
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ACID
 				duration = 1
@@ -1356,12 +1356,12 @@ COPY_EXISTING	~spwi615.spl~	override		// chain lighting
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spin596.spl~	override		// brown dragon acid breath
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ACID
 				duration = 1
@@ -1374,7 +1374,7 @@ COPY_EXISTING	~spin691.spl~	override		// black dragon breath
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ACID
 				duration = 1
@@ -1382,12 +1382,12 @@ COPY_EXISTING	~spin691.spl~	override		// black dragon breath
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 COPY_EXISTING	~spin708.spl~	override		// slime trap
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ACID
 				duration = 1
@@ -1400,7 +1400,7 @@ COPY_EXISTING	~spin913.spl~	override		// Mimic Acid
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ACID
 				duration = 1
@@ -1413,7 +1413,7 @@ COPY_EXISTING	~spin917.spl~	override		// Mane gas
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ACID
 				duration = 1
@@ -1426,7 +1426,7 @@ COPY_EXISTING	~spin917.spl~	override		// Mane gas
 //COPY_EXISTING ~spwi211.spl~ ~override~
 //    LPF ADD_SPELL_EFFECT INT_VAR
 //	      insert_point = 0
-//	      opcode = 324
+//	      opcode = 318
 //	      target = 2
 //	      parameter2 = ACID
 //	      duration = 1
@@ -1434,12 +1434,12 @@ COPY_EXISTING	~spin917.spl~	override		// Mane gas
 //        resource = EVAL ~%SOURCE_RES%~
 //     END
 //BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~%spwi427%.spl~	override		// vitriolic sphere
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ACID
 				duration = 1
@@ -1447,12 +1447,12 @@ COPY_EXISTING	~spin917.spl~	override		// Mane gas
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~dvwi526d.spl~	override		// mestil
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ACID
 				duration = 1
@@ -1465,7 +1465,7 @@ COPY_EXISTING	~spwi614.spl~	override		// death fog - it will no longer kill summ
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = ACID
 				duration = 1
@@ -1475,14 +1475,14 @@ COPY_EXISTING	~spwi614.spl~	override		// death fog - it will no longer kill summ
 	BUT_ONLY IF_EXISTS
 
 
-///  MAGIC DAMAGE IMMUNITY // 
+///  MAGIC DAMAGE IMMUNITY //
 
 
-COPY_EXISTING	~keldorn.spl~	override		// keldorn's armor 
+COPY_EXISTING	~keldorn.spl~	override		// keldorn's armor
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1490,12 +1490,12 @@ COPY_EXISTING	~keldorn.spl~	override		// keldorn's armor
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spin202.spl~	override		// wounds
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1508,7 +1508,7 @@ COPY_EXISTING	~keldorn.spl~	override		// keldorn's armor
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1516,12 +1516,12 @@ COPY_EXISTING	~keldorn.spl~	override		// keldorn's armor
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spin958.spl~	override		// wounds
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1529,12 +1529,12 @@ COPY_EXISTING	~keldorn.spl~	override		// keldorn's armor
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~spin986.spl~	override		// wounds
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1542,13 +1542,13 @@ COPY_EXISTING	~keldorn.spl~	override		// keldorn's armor
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
-	
+
+
 COPY_EXISTING	~spin564.spl~	override		// skull trap exp
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1561,7 +1561,7 @@ COPY_EXISTING	~spin912.spl~	override		// psi detonate
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1574,7 +1574,7 @@ COPY_EXISTING	~spin962.spl~	override		// beholder magic missile
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1587,7 +1587,7 @@ COPY_EXISTING	~sppr313.spl~	override		// holy  smite
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1595,12 +1595,12 @@ COPY_EXISTING	~sppr313.spl~	override		// holy  smite
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~sppr314.spl~	override		// blight
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1614,7 +1614,7 @@ COPY_EXISTING	~sppr612.spl~	override		// bolt of glory
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1623,12 +1623,12 @@ COPY_EXISTING	~sppr612.spl~	override		// bolt of glory
 		END
 	BUT_ONLY IF_EXISTS
 
- 
+
 COPY_EXISTING	~spwi003.spl~	override		// mm trap
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1641,7 +1641,7 @@ COPY_EXISTING	~spwi112.spl~	override		//mm trap
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1651,7 +1651,7 @@ COPY_EXISTING	~spwi112.spl~	override		//mm trap
 	BUT_ONLY IF_EXISTS
 
 //ACTION_IF FILE_EXISTS_IN_GAME ~spwi119a.spl~ THEN BEGIN // This is LMD. AAAAAAAARGH!
-/*COPY_EXISTING ~spwi119a.spl~ ~override~  
+/*COPY_EXISTING ~spwi119a.spl~ ~override~
 LPF ADD_SPELL_EFFECT
 	INT_VAR
 	opcode			= 324
@@ -1672,17 +1672,17 @@ COPY_EXISTING	~spwi314.spl~	override		// skull trap
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
 			STR_VAR
 				resource = EVAL ~%SOURCE_RES%~
 		END
-	BUT_ONLY IF_EXISTS 
+	BUT_ONLY IF_EXISTS
 
 //ACTION_IF FILE_EXISTS_IN_GAME ~spwi314.spl~ THEN BEGIN  // Vampiric Touch. AAAAAAAARGH
-/*COPY_EXISTING ~spwi314.spl~ ~override~  
+/*COPY_EXISTING ~spwi314.spl~ ~override~
 LPF ADD_SPELL_EFFECT
 	INT_VAR
 	opcode			= 324
@@ -1703,7 +1703,7 @@ COPY_EXISTING	~spwi616.spl~	override		// disintegrate
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1716,7 +1716,7 @@ COPY_EXISTING	~spwi812.spl~	override		// adhw
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1729,7 +1729,7 @@ COPY_EXISTING	~spwi950.spl~	override		// skull trap
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1742,7 +1742,7 @@ COPY_EXISTING	~spwi953.spl~	override		// dradael blue firaball
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1755,7 +1755,7 @@ COPY_EXISTING	~spwish32.spl~	override		// Wish ADHW
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = MAGIC
 				duration = 1
@@ -1770,7 +1770,7 @@ COPY_EXISTING	~bdmucou2.spl~	override		// mucus
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = POISN
 				duration = 1
@@ -1778,12 +1778,12 @@ COPY_EXISTING	~bdmucou2.spl~	override		// mucus
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 	COPY_EXISTING	~drgrbrht.spl~	override		// poison breath
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = POISN
 				duration = 1
@@ -1796,7 +1796,7 @@ COPY_EXISTING	~bdmucou2.spl~	override		// mucus
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = POISN
 				duration = 1
@@ -1810,7 +1810,7 @@ COPY_EXISTING	~bdmucou2.spl~	override		// mucus
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = POISN
 				duration = 1
@@ -1818,12 +1818,12 @@ COPY_EXISTING	~bdmucou2.spl~	override		// mucus
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 		COPY_EXISTING	~spin568.spl~	override		// mimic poison
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = POISN
 				duration = 1
@@ -1838,7 +1838,7 @@ COPY_EXISTING	~bdmucou2.spl~	override		// mucus
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = POISN
 				duration = 1
@@ -1851,7 +1851,7 @@ COPY_EXISTING	~bdmucou2.spl~	override		// mucus
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = POISN
 				duration = 1
@@ -1864,7 +1864,7 @@ COPY_EXISTING	~bdmucou2.spl~	override		// mucus
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = POISN
 				duration = 1
@@ -1877,7 +1877,7 @@ COPY_EXISTING	~bdmucou2.spl~	override		// mucus
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = POISN
 				duration = 1
@@ -1885,12 +1885,12 @@ COPY_EXISTING	~bdmucou2.spl~	override		// mucus
 				resource = EVAL ~%SOURCE_RES%~
 		END
 	BUT_ONLY IF_EXISTS
-	
+
 		COPY_EXISTING	~spwi016.spl~	override		// cloudkill trap
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = POISN
 				duration = 1
@@ -1905,7 +1905,7 @@ COPY_EXISTING	~bdmucou2.spl~	override		// mucus
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = POISN
 				duration = 1
@@ -1918,7 +1918,7 @@ COPY_EXISTING	~bdmucou2.spl~	override		// mucus
 		LPF	ADD_SPELL_EFFECT
 			INT_VAR
 				insert_point = 0
-				opcode = 324
+				opcode = 318
 				target = 2
 				parameter2 = POISN
 				duration = 1
