--- faiths_and_powers\language\french\new_spells.tra
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\faiths_and_powers\\language\\french\\new_spells.tra"	
@@ -318,14 +318,14 @@ Comme les pièges du voleur, ce sort échoue s'il est lancé en présence d'enne
 @51503 = ~Poing de pierre
 (Altération)
 
-Niveau : 2
-Sphère : Élémentaire (terre)
-Portée : le lanceur
-Durée : 1 tour
-Temps d'incantation : 3
-Zone d'effet : le lanceur
-
-Quand le lanceur jette ce sort, ses mains se transforment en blocs de pierre. Il peut attaquer au corps à corps et ses mains infligent 2d8 points de dégâts auxquels s'ajoute les dégâts dus à la force. De plus il est imprégné du pouvoir de la terre et sa force est augmentée de 4 points pour la durée du sort.~
+Niveau : 2
+Sphère : Élémentaire (terre)
+Portée : le lanceur
+Durée : 1 tour
+Temps d'incantation : 3
+Zone d'effet : le lanceur
+
+Quand le lanceur jette ce sort, ses mains se transforment en blocs de pierre. Il peut attaquer au corps à corps et ses mains infligent 2d8 points de dégâts auxquels s'ajoute les dégâts dus à la force. De plus il est imprégné du pouvoir de la terre et sa force est augmentée de 4 points pour la durée du sort.~
 
 @51504 = ~Poing de pierre
 
