--- SubtleD_Item_Tweaks\SubtleD_Item_Tweaks.tp2
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\SubtleD_Item_Tweaks\\SubtleD_Item_Tweaks.tp2"	
@@ -1,7 +1,7 @@
 BACKUP ~weidu_external/backup/SubtleD_Item_Tweaks~
 AUTHOR ~SubtleD~
 
-VERSION ~v1.3.3~
+VERSION ~v1.3.3 BWS-FR Fix~
 
 ALWAYS
 
@@ -20,6 +20,11 @@ LANGUAGE
   ~english~
   ~SubtleD_Item_Tweaks/lang/en_US/setup.tra~
 
+LANGUAGE
+  ~Francais~
+  ~french~
+  ~SubtleD_Item_Tweaks/lang/en_US/setup.tra~
+  ~SubtleD_Item_Tweaks/lang/fr_FR/setup.tra~
 
 //__________________________________________________________________________________
 //__________________________________________________________________________________
