--- item_rev\components\kensai_use_bracers.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\item_rev\\components\\kensai_use_bracers.tpa"	
@@ -6,15 +6,14 @@ INCLUDE ~item_rev/lib/usability_description.tpa~
 //  kit description
 
 COPY_EXISTING kitlist.2da override
-  READ_2DA_ENTRY 3 4 9 desc
-  SET_2DA_ENTRY 3 4 9 ~kensai_bracer~
+  READ_2DA_ENTRY 4 4 9 desc
   GET_STRREF desc kensai
   INNER_PATCH_SAVE bracer ~%kensai%~ BEGIN
     SPRINT t1 @100413
     SPRINT t2 @100414
     REPLACE_TEXTUALLY ~%t1%~ ~%t2%~
   END
-  REPLACE ~kensai_bracer~ ~%bracer%~
+  STRING_SET_EVALUATE desc ~%bracer%~
 
 //////////////////////////////////////////////
 //  descriptions
