--- edwin\tra\french\setup.tra
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\edwin\\setup.tra"
@@ -1,6 +1,7 @@
 // -*- coding: utf-8 -*-
 //This file is encoded in UTF-8
 
+@0    = ~La Romance d'Edwin~
 @50000    = ~La Romance d'Edwin~
 @31   = ~La romance d'Edwin n'est pas compatible avec le composant "Portrait feminin pour Edwina" du BG2 Tweak Pack. Il doit etre desinstalle afin de pouvoir installer la Romance d'Edwin.~
 @50002   = ~Restaurer le portrait BGI d'Edwin~
