--- stratagems\priest\spellchoices_defensive\demivrgvs\druid.tph
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\druid.tph"	
@@ -38,7 +38,7 @@ BEGIN
       7 BEGIN
          PUSH spells ~BLESS FLAME_BLADE BARKSKIN~
          PUSH_RANDOM spells (CURE_DISEASE null)
-         PUSH_RANDOM spells (DEFENSIVE_HARMONY null
+         PUSH_RANDOM spells (DEFENSIVE_HARMONY null)
       END
 /////////////////////////////////////////// level 8 /////////////////////////////////////
       8 BEGIN
