--- stratagems\fiend\ssl\summon\dw#hlbal.ssl
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\dw#hlbal.ssl"	
@@ -106,7 +106,7 @@ END
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	TriggerBlock(Insane)
-	!GlobalGT("chapter","GLOBAL",7)
+	!GlobalGT("chapter","GLOBAL",%bg2_chapter_7%)
 	!Gender(scstarget,SUMMONED)
 	!Gender(scstarget,20)
 	TriggerBlock(SpellDeflect|Enemy|Helpless)
@@ -117,7 +117,7 @@ END
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	TriggerBlock(HardPlus)
-	GlobalGT("chapter","GLOBAL",7)
+	GlobalGT("chapter","GLOBAL",%bg2_chapter_7%)
 	!Gender(scstarget,SUMMONED)
 	!Gender(scstarget,20)
 	TriggerBlock(SpellDeflect|Enemy|Helpless)
