--- stratagems\fiend\ssl\dw#llbal.ssl
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\dw#llbal.ssl"	
@@ -105,7 +105,7 @@ END
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	TriggerBlock(Insane)
-	!GlobalGT("chapter","GLOBAL",7)
+	!GlobalGT("chapter","GLOBAL",%bg2_chapter_7%)
 	!Gender(scstarget,SUMMONED)
 	!Gender(scstarget,20)
 	TriggerBlock(SpellTurn|Helpless)
@@ -116,7 +116,7 @@ END
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	TriggerBlock(HardPlus)
-	GlobalGT("chapter","GLOBAL",7)
+	GlobalGT("chapter","GLOBAL",%bg2_chapter_7%)
 	!Gender(scstarget,SUMMONED)
 	!Gender(scstarget,20)
 	TriggerBlock(SpellTurn|Helpless)
