--- stratagems\tactical_bg2\ssl\dw#random.ssl
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\dw#random.ssl"	
@@ -6,7 +6,7 @@ IF
 	OnCreation()
 	GlobalGT("RandomEncounters","GLOBAL",1)
 	Global("DMWWRandomTravelers","GLOBAL",0)
-	GlobalGT("chapter","GLOBAL",5)
+	GlobalGT("chapter","GLOBAL",%bg2_chapter_5%)
 THEN 
 	RESPONSE #100
 		SetGlobal("DMWWRandomTravelers","GLOBAL",1)
@@ -31,7 +31,7 @@ IF TRIGGER
 		PartyHasItem("Blun30c")
 		PartyHasItem("Blun30d")
 		PartyHasItem("Blun30")
-	GlobalGT("chapter","GLOBAL",5)
+	GlobalGT("chapter","GLOBAL",%bg2_chapter_5%)
 	Global("DMWWRakshasaFlail","GLOBAL",0)
 	!GlobalTimerNotExpired("DMWWRakshasaTimer","GLOBAL")
 THEN DO
@@ -56,7 +56,7 @@ IF TRIGGER
 		PartyHasItem("Blun30c")
 		PartyHasItem("Blun30d")
 		PartyHasItem("Blun30")
-	GlobalGT("chapter","GLOBAL",5)
+	GlobalGT("chapter","GLOBAL",%bg2_chapter_5%)
 	XPGT(Player1,1500000)
 	Global("DMWWRakshasaFlail","GLOBAL",1)
 	!GlobalTimerNotExpired("DMWWRakshasaTimer","GLOBAL")
