--- stratagems\ai\ssl\hama.ssl
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\hama.ssl"	
@@ -173,7 +173,7 @@ IF
   !GlobalTimerNotExpired("teleporttimer","LOCALS")
   !StateCheck(Myself,STATE_REALLY_DEAD)
   OR(3)
-       RandomNum(1,2)
+       RandomNum(2,1)
        Range(NearestEnemyOf(Myself),10)
        HPPercentLT(Myself,60)
 
@@ -217,7 +217,7 @@ IF
   !GlobalTimerNotExpired("teleporttimer","LOCALS")
   !StateCheck(Myself,STATE_REALLY_DEAD)
   OR(3)
-       RandomNum(1,2)
+       RandomNum(2,1)
        Range(NearestEnemyOf(Myself),10)
        HPPercentLT(Myself,60)
 
@@ -255,7 +255,7 @@ IF
   !GlobalTimerNotExpired("teleporttimer","LOCALS")
   !StateCheck(Myself,STATE_REALLY_DEAD)
   OR(3)
-       RandomNum(1,2)
+       RandomNum(2,1)
        Range(NearestEnemyOf(Myself),10)
        HPPercentLT(Myself,60)
 
@@ -294,7 +294,7 @@ IF
   !GlobalTimerNotExpired("teleporttimer","LOCALS")
   !StateCheck(Myself,STATE_REALLY_DEAD)
   OR(3)
-       RandomNum(1,2)
+       RandomNum(2,1)
        Range(NearestEnemyOf(Myself),10)
        HPPercentLT(Myself,60)
 
@@ -333,7 +333,7 @@ IF
   !GlobalTimerNotExpired("teleporttimer","LOCALS")
   !StateCheck(Myself,STATE_REALLY_DEAD)
   OR(3)
-       RandomNum(1,2)
+       RandomNum(2,1)
        Range(NearestEnemyOf(Myself),10)
        HPPercentLT(Myself,60)
 
@@ -372,7 +372,7 @@ IF
   !GlobalTimerNotExpired("teleporttimer","LOCALS")
   !StateCheck(Myself,STATE_REALLY_DEAD)
   OR(3)
-       RandomNum(1,2)
+       RandomNum(2,1)
        Range(NearestEnemyOf(Myself),10)
        HPPercentLT(Myself,60)
 
@@ -410,7 +410,7 @@ IF
   !GlobalTimerNotExpired("teleporttimer","LOCALS")
   !StateCheck(Myself,STATE_REALLY_DEAD)
   OR(3)
-       RandomNum(1,2)
+       RandomNum(2,1)
        Range(NearestEnemyOf(Myself),10)
        HPPercentLT(Myself,60)
 
