--- stratagems\gameplay\hla_innate.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\hla_innate.tpa"	
@@ -102,10 +102,10 @@ BEGIN
 				WRITE_LONG 0x34 1 // we need this on vanilla, otherwise HaveSpell() causes crashes. Probably it's safe in EE too, but this component has broken so many times that I don't want to risk it!
 			 END
          BUT_ONLY
-         ACTION_FOR_EACH copy_spell IN projimag simualcr BEGIN
+         ACTION_FOR_EACH copy_spell IN projimag simulacr BEGIN
             ACTION_IF FILE_EXISTS_IN_GAME "%copy_spell%.spl" BEGIN
                COPY_EXISTING "%copy_spell%.spl" override
-                   LPF ADD_SPELL_EFFECT INT_VAR opcode=172 target=1 timing=1 STR_VAR resource="%resref_new%" END
+                   LPF ADD_SPELL_EFFECT INT_VAR opcode=172 target=1 timing=1 STR_VAR resource="%new_resref%" END
                BUT_ONLY
             END
          END
