--- stratagems\genai\ssl\potionuse.ssl
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\potionuse.ssl"	
@@ -45,8 +45,8 @@ THEN DO
         Action(SpellMyself,MONK_LAY_ON_HANDS)
 	Action(Potion,"dw#ptn55",@101055)
 	Action(Potion,"potn55",@101055)
-	ActionCondition(Potion,"dw#ptn52",@101052;!GlobalGT("Chapter","GLOBAL",7))
-	ActionCondition(Potion,"potn52",@101052;!GlobalGT("Chapter","GLOBAL",7))
+	ActionCondition(Potion,"dw#ptn52",@101052;!GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%))
+	ActionCondition(Potion,"potn52",@101052;!GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%))
 END
 
 IF TRIGGER
