--- stratagems\setup-stratagems.tp2
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\setup-stratagems.tp2"	
@@ -1,7 +1,7 @@
 BACKUP ~weidu_external/backup/stratagems~
 AUTHOR ~For help troubleshooting installation problems, go to the Sword Coast Stratagems forum at forums.gibberlings3.net.~
 // MODDER setup_tra none area_variables none missing_extern none missing_resref none ict2_actions none missing_eval none overwriting_file none fun_args warn
-VERSION ~34.3~
+VERSION ~34.3 BWS-FR Fix~
 README ~stratagems/doc/readme-stratagems.html~
 AUTO_EVAL_STRINGS
 
@@ -54,7 +54,7 @@ GERMAN
  ~weidu_external/lang/stratagems/german/setup.tra~
 
 LANGUAGE
-"Francais (Mornagest/Isaya/Graoumf/Gwendolyne/Jazira) - last updated version 34.2"
+"Francais (Mornagest/Isaya/Graoumf/Gwendolyne/Jazira) - last updated version 34.3"
 FRENCH
  ~stratagems/lang/english/setup.tra~
  ~weidu_external/lang/stratagems/english/setup.tra~
