--- stratagems\gameplay\skip_candlekeep.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\gameplay\\skip_candlekeep.tpa"
@@ -18,15 +18,19 @@ DEFINE_ACTION_FUNCTION candlekeep_no_tutors BEGIN
     LAF get_area_script STR_VAR area= ~%Candlekeep%~ RET script END

     ACTION_IF enhanced_edition BEGIN
-      OUTER_SPRINT swapline ~DisplayString(Myself,31107)~
+      ACTION_IF is_eet BEGIN
+       OUTER_SPRINT swapline ~DisplayString(Myself,231107)=>null DisplayString(Myself,216214)=>null~
+      END ELSE BEGIN
+       OUTER_SPRINT swapline ~DisplayString(Myself,31107)=>null DisplayString(Myself,16214)=>null~
+      END
     END ELSE BEGIN
       ACTION_IF is_tutu BEGIN
-       OUTER_SPRINT ~swapline~ ~DisplayStringHead(Myself,86017)~
+       OUTER_SPRINT swapline ~DisplayStringHead(Myself,86017)=>null~
       END ELSE BEGIN
-       OUTER_SPRINT ~swapline~ ~DisplayStringHead(Myself,16214)~
+       OUTER_SPRINT swapline ~DisplayStringHead(Myself,16214)=>null~
       END
     END
-    LAF swap_text STR_VAR files= ~%script%.bcs~ swaps= ~%swapline%=>null~ END
+    LAF swap_text STR_VAR files= ~%script%.bcs~ swaps=~%swapline%~ END
     ACTION_IF FILE_EXISTS_IN_GAME "macinit.bcs" BEGIN
        LAF swap_text STR_VAR files=macinit.bcs swaps=~SetGlobal("MACGameInit","GLOBAL",1)=>SetGlobal("MACGameInit","GLOBAL",1)EscapeArea()~ END
     END