--- stratagems\gameplay\spellhold_cost.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\spellhold_cost.tpa"	
@@ -20,6 +20,8 @@ BEGIN
 		  OUTER_PATCH_SAVE string "%string%" BEGIN
 			REPLACE_TEXTUALLY "20\([, \.]?\)\(000\)" "%asking_price_thousands%\1\2"
 			REPLACE_TEXTUALLY "15\([, \.]?\)\(000\)" "%actual_price_thousands%\1\2"
+			REPLACE_TEXTUALLY "20 000" "%asking_price_thousands% 000"
+			REPLACE_TEXTUALLY "15 000" "%actual_price_thousands% 000"
 		  END
 		  STRING_SET_EVALUATE tlk_entry "%string%"
     END
@@ -137,6 +139,8 @@ DEFINE_ACTION_MACRO ~spellholdcost_mod_compatibility~ BEGIN
 				REPLACE_TEXTUALLY ~15,000~ ~%actual_price%~
 				REPLACE_TEXTUALLY ~20000~ ~%original_price%~
 				REPLACE_TEXTUALLY ~15000~ ~%actual_price%~
+				REPLACE_TEXTUALLY ~20 000~ ~%original_price%~
+				REPLACE_TEXTUALLY ~15 000~ ~%actual_price%~
 				SPRINT ~15K_text~ @1601
 				SPRINT ~20K_text~ @1602
 				REPLACE_TEXTUALLY CASE_INSENSITIVE ~%15K_text%~ ~%actual_price%~
