--- stratagems\werewolf\werewolf.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\werewolf.tpa"	
@@ -132,7 +132,7 @@ END
 DEFINE_ACTION_FUNCTION kaishas_confrontation BEGIN
 
  ACTION_IF !FILE_EXISTS_IN_GAME "dw#hlpka.bcs" BEGIN // make sure the Calling version isn't already installed
-   ADD_JOURNAL @21648 @22000
+   ADD_JOURNAL EXISTING @21614 @21648 @22000
    ACTION_IF GAME_IS ~bgee eet~ BEGIN // lowlevel check to hold onto Calling interoperability
     OUTER_SPRINT ee_erase_string "EraseJournalEntry(26907)
 EraseJournalEntry(26908)
