--- EET\tbl\compatibility.tbl
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\EET\\compatibility.tbl"
@@ -4,6 +4,7 @@ BWS-URL 0            //dummy entry from Big World Setup
 DLCMERGER 0          //DLC Merger
 BGEETEXTPACK 0       //RUS BG1
 SODRUS 0             //RUS SOD
+CORRECTFRBG1EE 0     //FR BG1
 
 BG1AERIE 1           //Aerie for BG:EE
 BG1NPC 1             //The BG1 NPC Project
