--- themed_tweaks\dialogues\bg2_snark.d
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\themed_tweaks\\dialogues\\bg2_snark.d"	
@@ -4,6 +4,14 @@
 /////////////////////////////////////////////////////////////
 
 EXTEND_BOTTOM SUELLE2 18
-	IF ~ReputationLT(Player1,10) !Global("#L_Snark","GLOBAL",0)~ THEN REPLY @3000 /* ~Only partially?~ */ + 19
-	IF ~ReputationGT(Player1,9) !Global("#L_Snark","GLOBAL",0)~ THEN REPLY @3000 /* ~Only partially?~ */ + 20
-END
+	IF ~ReputationLT(Player1,10) !Global("#L_Snark","GLOBAL",0)~ THEN REPLY @3000 /* ~Only partially?~ */ DO ~SetGlobal("soa_complete","GLOBAL",1)
+ClearAllActions()
+StartCutSceneMode()
+StartCutSceneEx("cut100a",FALSE)
+~ EXIT
+	IF ~ReputationGT(Player1,9) !Global("#L_Snark","GLOBAL",0)~ THEN REPLY @3000 /* ~Only partially?~ */ DO ~SetGlobal("soa_complete","GLOBAL",1)
+ClearAllActions()
+StartCutSceneMode()
+StartCutSceneEx("cut100b",FALSE)
+~ EXIT
+END
\ No newline at end of file
