--- themed_tweaks\themed_tweaks.tp2
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\themed_tweaks\\themed_tweaks.tp2"	
@@ -1,6 +1,6 @@
 BACKUP ~weidu_external/backup/themed_tweaks~
 SUPPORT ~https://www.gibberlings3.net/forums/topic/32346-themed-tweaks-for-bgee-sod-bg2ee-tob-eet/~
-VERSION ~v0.4~
+VERSION ~v0.4 BWS-FR Fix~
 
 README ~themed_tweaks/readme.themed_tweaks.%LANGUAGE%.txt~ ~themed_tweaks/readme.themed_tweaks.english.txt~
 
