--- questpack\tp2\ds.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\questpack\\tp2\\ds.tpa"	
@@ -4,7 +4,6 @@ ACTION_IF NOT FILE_EXISTS_IN_GAME ~fl#qpds.mrk~ BEGIN
   PRINT ~Initializing Detectable Spells installation ...~                    // let the user know that DS are being installed
   OUTER_SPRINT path ~%MOD_FOLDER%/tp2/ds~
   INCLUDE ~%path%/ds.tph~
-  LAF detectable_spells_setup STR_VAR path END
-  LAF detectable_spells STR_VAR path table = ~cddetect.2da~ END
+  LAF detectable_spells END
   COPY_EXISTING ~sw1h01.itm~ ~override/fl#qpds.mrk~
 END
