--- CtB\scripts\ctb\CBBDenem.BAF
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\CtB\\scripts\\ctb\\CBBDenem.BAF"	
@@ -87,11 +87,10 @@ THEN
 END
 
 IF
-	OR(5)
+	OR(4)
 		SpellCast([GOODCUTOFF],WIZARD_CLOUDKILL)
 		SpellCast([GOODCUTOFF],WIZARD_INCENDIARY_CLOUD)
 		SpellCast([GOODCUTOFF],WIZARD_DEATH_FOG)
-		SpellCast([GOODCUTOFF],WIZARD_FOG_CLOUD)
 		SpellCast([GOODCUTOFF],WIZARD_STINKING_CLOUD)
 	!GlobalTimerNotExpired("WINGBUFFET","AR3521")
 THEN
