--- CtB\scripts\ctb\CBBDAWEN.BAF
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\CtB\\scripts\\ctb\\CBBDAWEN.BAF"	
@@ -96,11 +96,10 @@ THEN
 END
 
 IF
-	OR(5)
+	OR(4)
 		SpellCast([GOODCUTOFF],WIZARD_CLOUDKILL)
 		SpellCast([GOODCUTOFF],WIZARD_INCENDIARY_CLOUD)
 		SpellCast([GOODCUTOFF],WIZARD_DEATH_FOG)
-		SpellCast([GOODCUTOFF],WIZARD_FOG_CLOUD)
 		SpellCast([GOODCUTOFF],WIZARD_STINKING_CLOUD)
 	!GlobalTimerNotExpired("WINGBUFFET","LOCALS")
 THEN
