--- bg1re\tra\francais\c#eltan.tra
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\bg1re\\c#eltan.tra"	
@@ -370,6 +370,6 @@ Eltan m'a dit qu'on avait trouv
 @327  = ~Ecoutez-moi bien : maintenant il faut que vous partiez. Désolé d'être un peu rude, mais le souvenir de cette nuit m'emplit encore de frayeur. Je suis sûr que la personne qui a fait assassiner La Balafre n'aurait aucun problème à éliminer deux autres témoins. Je vous promets d'informer le duc Eltan dès votre départ.~
 
 /* new for v10.0 */
-@328  = ~I've taken care of the assassins.  I'm just waiting for the coronation to start so I can confront Sarevok in front of witnesses.~
-@329  = ~I've ruined Sarevok's plans to murder the other dukes. Don't worry, they're safe.~
-@330  = ~Good, that's good.  I can rest easier now.~
+@328 = ~J'ai réglé leur compte aux assassins. J'attends maintenant le début du couronnement pour faire face à Sarevok devant témoins.~
+@329 = ~J'ai déjoué les plans de Sarevok visant à assassiner les autres ducs. Ne vous inquiétez pas, ils sont en sécurité.~
+@330 = ~Bien, très bien. Je vais pouvoir dormir plus tranquille maintenant.~
