--- imnesvale\scripts\easy\duarim.baf
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\imnesvale\\scripts\\easy\\duarim.baf"	
@@ -13,7 +13,6 @@ THEN
 		ReallyForceSpell(Myself,WIZARD_STONE_SKIN)
 		ReallyForceSpell(Myself,WIZARD_MIRROR_IMAGE)
 		ReallyForceSpell(Myself,WIZARD_BLUR)
-		ReallyForceSpell(Myself,CLERIC_RESIST_COLD)
 		ReallyForceSpell(Myself,CLERIC_RESIST_FIRE)
 		ReallyForceSpell(Myself,CLERIC_CHAOTIC_COMMANDS)
 		ReallyForceSpell(Myself,WIZARD_PROTECTION_FROM_ENERGY)
@@ -54,7 +53,6 @@ THEN
 		ReallyForceSpell(Myself,WIZARD_STONE_SKIN)
 		ReallyForceSpell(Myself,WIZARD_MIRROR_IMAGE)
 		ReallyForceSpell(Myself,WIZARD_BLUR)
-		ReallyForceSpell(Myself,CLERIC_RESIST_COLD)
 		ReallyForceSpell(Myself,CLERIC_RESIST_FIRE)
 		ReallyForceSpell(Myself,CLERIC_CHAOTIC_COMMANDS)
 		ReallyForceSpell(Myself,WIZARD_PROTECTION_FROM_ENERGY)
@@ -107,7 +105,6 @@ THEN
 		ReallyForceSpell(Myself,WIZARD_STONE_SKIN)
 		ReallyForceSpell(Myself,WIZARD_MIRROR_IMAGE)
 		ReallyForceSpell(Myself,WIZARD_BLUR)
-		ReallyForceSpell(Myself,CLERIC_RESIST_COLD)
 		ReallyForceSpell(Myself,CLERIC_RESIST_FIRE)
 		ReallyForceSpell(Myself,CLERIC_CHAOTIC_COMMANDS)
 		ReallyForceSpell(Myself,WIZARD_PROTECTION_FROM_ENERGY)
