--- atweaks\baf\fey\rr#snymp.baf
+++ "b/E:\\GOG\\Baldur's Gate II - Enhanced Edition - Test\\atweaks\\baf\\fey\\rr#snymp.baf"	
@@ -1338,7 +1338,7 @@ IF
 	CheckStatLT(Myself,50,SPELLFAILUREPRIEST)
 	!StateCheck(Myself,STATE_SILENCED)
 	!CheckStatGT(Myself,0,CLERIC_INSECT_PLAGUE)
-	See(EigthNearestEnemyOfType([0.HUMANOID]))
+	See(EighthNearestEnemyOfType([0.HUMANOID]))
 	!Class(LastSeenBy(Myself),MONK)
         !Kit(LastSeenBy(Myself),INQUISITOR)
         !Kit(LastSeenBy(Myself),CAVALIER)
