--- atweaks\setup-atweaks.tp2
+++ "b/E:\\GOG\\Baldur's Gate II - Enhanced Edition - Test\\atweaks\\setup-atweaks.tp2"	
@@ -8,7 +8,7 @@
 
 BACKUP ~atweaks/backup~
 AUTHOR "Wisp, shsforums.net"                                                       // Author and e-mail address
-VERSION ~v4.53~                                                                    // Mod version
+VERSION ~v4.53 BWS-FR Fix~                                                         // Mod version
 
 ALWAYS                                                                             // Always do this block
   INCLUDE ~aTweaks/LIB/RR#AGLEF.TPH~                                               // Include my custom "Add Global Effect" function
