--- BridgesBlock\dialogue\L#NWLIB.d
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\BridgesBlock\\dialogue\\L#NWLIB.d"	
@@ -63,7 +63,7 @@ CHAIN L#NWLIB 01.Q21
 == AerieJ IF ~InParty("aerie") !StateCheck("aerie",CD_STATE_NOTVALID)~ THEN @29
 == NaliaJ IF ~InParty("nalia") !StateCheck("nalia",CD_STATE_NOTVALID)~ THEN @30
 == KorganJ IF ~InParty("korgan") !StateCheck("korgan",CD_STATE_NOTVALID)~ THEN @31
-== KeldorJ IF ~InParty("keldorn") !StateCheck("kelodrn",CD_STATE_NOTVALID)~ THEN @32
+== KeldorJ IF ~InParty("keldorn") !StateCheck("keldorn",CD_STATE_NOTVALID)~ THEN @32
 == YoshJ IF ~InParty("yoshimo") !StateCheck("yoshimo",CD_STATE_NOTVALID)~ THEN @33
 ==L#NWLIB @34
 END
