--- AC_QUEST\tpa\hopsi.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\AC_QUEST\\tpa\\hopsi.tpa"	
@@ -12,10 +12,10 @@ SAY SELECT_COMMON1 @80018
 COPY ~AC_QUEST/cre/ACQ13002.CRE~ ~override~
 SAY NAME1 @23
 SAY NAME2 @23
-PATCH_IF GAME_IS ~bgt~ THEN BEGIN
+PATCH_IF GAME_IS ~bgt eet~ THEN BEGIN
 ADD_CRE_ITEM ~RNDTRE08~ #0 #0 #0 ~NONE~ ~INV~
 END
-PATCH_IF GAME_IS ~bgee eet tutu_totsc~ THEN BEGIN
+PATCH_IF GAME_IS ~bgee tutu_totsc~ THEN BEGIN
 ADD_CRE_ITEM ~RNDTRE04~ #0 #0 #0 ~NONE~ ~INV~
 END
 
