--- iepbanters\lib\iepbanters_accelerator.tpa	
+++ "b/e:\\Jeux modd\303\251s\\Baldur's Gate 2\\BigWorldSetupFr\\OverwriteFiles\\BWP\\iepbanters\\lib\\iepbanters_accelerator.tpa"	
@@ -9,7 +9,7 @@
 // -----------------------------------------------------------
 UNINSTALL setup-banterpack.tp2 1
 UNINSTALL setup-banterpack.tp2 3
-UNINSTALL setup-bg1npc.tp2 25
+UNINSTALL setup-bg1npc.tp2 100
 UNINSTALL Kelsey.tp2 3
 UNINSTALL Keto.tp2 2
 
