--- bg1ub\scar\ubscar.d
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\bg1ub\\ubscar.d"	
@@ -1,14 +1,7 @@
 
 REPLACE_STATE_TRIGGER %tutu_var%ARKION 6 ~Global("HelpArkion","GLOBAL",1)~
 
-
 APPEND %tutu_var%ARKION
-IF WEIGHT #0 ~Global("HelpArkion","GLOBAL",0)
-PartyHasItem("%tutu_scriptbg%MISC79")~ THEN BEGIN EitherBody
-  SAY @0
-  IF ~PartyHasItem("%tutu_scriptbg%MISC79")~ THEN DO ~SetGlobal("HelpArkion","GLOBAL",1) TakePartyItem("%tutu_scriptbg%MISC79") GivePartyGold(250) AddexperienceParty(1800)~ EXIT
-END
-
 IF WEIGHT #0 ~Global("HelpArkion","GLOBAL",0)
 PartyHasItem("UBFEBODY")~ THEN BEGIN NobleBody
   SAY @16
