--- CtB\lib\cheat_var.tph
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\CtB\\cheat_var.tph"	
@@ -1,7 +1,7 @@
 ACTION_IF FILE_EXISTS_IN_GAME ~bgee.lua~ BEGIN
 	COPY_EXISTING ~bgee.lua~ ~override/bgee.lua~
 	REPLACE_TEXTUALLY
-~{"BD0010", "Test"},~
+~{"OH7200", "Hexxat - Bottom Right Quarter (Zakhara)"},~
 ~--CtB Areas
 
 	{"AR3520", "Stormhorns"},
@@ -114,7 +114,7 @@ ACTION_IF FILE_EXISTS_IN_GAME ~bgee.lua~ BEGIN
 	{"CB3667", "East Hlondeth: House Extaminos L0_7"},
 	{"CB3668", "East Hlondeth: House Extaminos L0_8"},
 		
-	{"BD0010", "Test"},
+	{"OH7200", "Hexxat - Bottom Right Quarter (Zakhara)"},
 ~
 
 END
