--- stratagems\tactical_bg2\ssl\c6_spawn_prototype.ssl
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\c6_spawn_prototype.ssl"	
@@ -1,7 +1,7 @@
 BEGIN_ACTION_DEFINITION
 	Name(SpawnGroupChapter)
 	TRIGGER
-	        GlobalGT("Chapter","GLOBAL",5)
+	        GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 	        Entered([ANYONE])
 	ACTION
 		RESPONSE #scsprob1
@@ -13,7 +13,7 @@ BEGIN_ACTION_DEFINITION
 	TRIGGER
 	        OR(2)
 	              XPGT(Player1,scsargument1)
-	              GlobalGT("Chapter","GLOBAL",5)
+	              GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 	        Entered([ANYONE])
 	ACTION
 		RESPONSE #scsprob1
