--- stratagems\fiend\ssl\dembal01.ssl
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\dembal01.ssl"	
@@ -164,7 +164,7 @@ IF TRIGGER
         IgnoreBlock(IsDemonLord)
 	TargetBlock(EnemiesInOrder)
 	TriggerBlock(Insane)
-	!GlobalGT("chapter","GLOBAL",7)
+	!GlobalGT("chapter","GLOBAL",%bg2_chapter_7%)
 	!Gender(scstarget,SUMMONED)
 	!Gender(scstarget,20)
 	TriggerBlock(SpellDeflect|Enemy|Helpless)
@@ -176,7 +176,7 @@ IF TRIGGER
         IgnoreBlock(IsDemonLord)
 	TargetBlock(EnemiesInOrder)
 	TriggerBlock(HardPlus)
-	GlobalGT("chapter","GLOBAL",7)
+	GlobalGT("chapter","GLOBAL",%bg2_chapter_7%)
 	!Gender(scstarget,SUMMONED)
 	!Gender(scstarget,20)
 	TriggerBlock(SpellTurn|Enemy|Helpless)
