--- stratagems\demilich\demilich.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\demilich\\demilich.tpa"
@@ -34,7 +34,7 @@ DEFINE_ACTION_FUNCTION demilich BEGIN

        // demiliches lose immunity to spells above L5

-       LAF edit_item STR_VAR item=demilich editstring=~delete_effect_global=>parameter1>5~ END
+       LAF edit_item STR_VAR item=demilich editstring=~delete_effect_global=>"opcode=102 and parameter1>5"~ END

        // if SR isn't installed, modify demilich imprisonment to grant a saving throw at -5