--- stratagems\firegiant\resource\dw#mkfgi.baf
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\dw#mkfgi.baf"	
@@ -1,5 +1,5 @@
 IF
-	RandomNum(1,6)
+	RandomNum(6,1)
 THEN
 	RESPONSE #100
 		CreateCreature("dw#hgwa1",[-1.-1],0)
@@ -7,7 +7,7 @@ THEN
 END
 
 IF
-	RandomNum(2,6)
+	RandomNum(6,2)
 THEN
 	RESPONSE #100
 		CreateCreature("dw#hgwa2",[-1.-1],0)
@@ -15,7 +15,7 @@ THEN
 END
 
 IF
-	RandomNum(3,6)
+	RandomNum(6,3)
 THEN
 	RESPONSE #100
 		CreateCreature("dw#hgwa3",[-1.-1],0)
@@ -23,7 +23,7 @@ THEN
 END
 
 IF
-	RandomNum(4,6)
+	RandomNum(6,4)
 THEN
 	RESPONSE #100
 		CreateCreature("dw#hgwa4",[-1.-1],0)
@@ -31,7 +31,7 @@ THEN
 END		
 
 IF
-	RandomNum(5,6)
+	RandomNum(6,6)
 THEN
 	RESPONSE #100
 		CreateCreature("dw#hgwa5",[-1.-1],0)
