--- stratagems\initial\immunities_via_324.tph
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\stratagems\\initial\\immunities_via_324.tph"	
@@ -65,7 +65,7 @@ BEGIN
 	12		76			FEEBLEMIND_IMMUNITY	
 	13		16			HASTE_IMMUNITY		
 	14		40			SLOW_IMMUNITY		
-	15		55			DEATH_IMMUNITY		
+	15		13			DEATH_IMMUNITY
 	16		minorglobe	MINOR_GLOBE			
 	17		majorglobe	MAJOR_GLOBE						
 	>>>>>>>>
@@ -153,8 +153,8 @@ BEGIN
 
 // do this here because we don't want repetitions in the immunity items
 
-	OUTER_SET $opcode_splstate_map("209")=$opcode_splstate_map("55") // Death Immunity protects from PW:Kill
-	OUTER_SET $opcode_splstate_map("13")=$opcode_splstate_map("55") // Death Immunity protects from 'kill target'
+	OUTER_SET $opcode_splstate_map("209")=$opcode_splstate_map("13") // Death Immunity protects from PW:Kill
+	OUTER_SET $opcode_splstate_map("55")=$opcode_splstate_map("13") // Death Immunity protects from 'kill target'
 	OUTER_SET $opcode_splstate_map("210")=$opcode_splstate_map("45") // Stun Immunity protects from PW:Stun
 	OUTER_SET $opcode_splstate_map("217")=$opcode_splstate_map("39") // Unconsciousness Immunity protects from PW:Sleep
 
