--- stratagems\sfo2e\lib_immunity.tph
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\stratagems\\sfo2e\\lib_immunity.tph"	
@@ -95,8 +95,8 @@ DEFINE_ACTION_MACRO immunity_effects_load BEGIN
 		icons=>"51"
 		animations=>""
 		display_icon=>"99"
-		immune_opcode=>55
-		extra_immune_opcode=>"13 209"
+		immune_opcode=>13
+		extra_immune_opcode=>"209"
 		splstate=>DEATH_IMMUNITY
 	END
 	
