--- stratagems\ssl\libdata.txt
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\stratagems\\ssl\\libdata.txt"	
@@ -5,7 +5,7 @@
 39 101 Sleep               SLEEP_IMMUNITY           *
 40 101 Slow                SLOW_IMMUNITY            *
 45 101 Stun                STUN_IMMUNITY            *
-55 101 Slay                DEATH_IMMUNITY           *
+13 101 Slay                DEATH_IMMUNITY           *
 128 101 ConfuseProof       CONFUSION_IMMUNITY       *
 134 101 Petrify            PETRIFY_IMMUNITY         *
 175 101 Hold               HOLD_IMMUNITY            *
