--- stratagems\gameplay\random_scrolls.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\random_scrolls.tpa"	
@@ -306,7 +306,9 @@ DEFINE_PATCH_FUNCTION ARE_patch_random_scroll BEGIN
 					rng=RANDOM (1 scroll_drop_fraction)
 					PATCH_IF rng=1 BEGIN
 						LPF determine_scroll_swap STR_VAR rndtre="%resref%" RET scroll_item END
-						WRITE_ASCIIE offset "%scroll_item%" (8)
+						PATCH_IF NOT "%scroll_item%" STRING_EQUAL_CASE "null" BEGIN
+							WRITE_ASCIIE offset "%scroll_item%" (8)
+						END
 					END
 			 END
 			 DEFAULT END
@@ -338,8 +340,10 @@ DEFINE_PATCH_FUNCTION CRE_patch_random_scroll BEGIN
 					rng=RANDOM (1 scroll_drop_fraction)
 					PATCH_IF rng=1 BEGIN
 						LPF determine_scroll_swap STR_VAR rndtre="%resref%" RET scroll_item END
-						WRITE_ASCIIE offset "%scroll_item%" (8)
-						WRITE_SHORT (offset+0xa) 0
+						PATCH_IF NOT "%scroll_item%" STRING_EQUAL_CASE "null" BEGIN
+							WRITE_ASCIIE offset "%scroll_item%" (8)
+							WRITE_SHORT (offset+0xa) 0
+						END
 						
 					END
               END
