--- stratagems\setup-stratagems.tp2
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\stratagems\\setup-stratagems.tp2"	
@@ -1,7 +1,7 @@
 BACKUP ~weidu_external/backup/stratagems~
 AUTHOR ~For help troubleshooting installation problems, go to the Sword Coast Stratagems forum at gibberlings3.net.~
 //MODDER setup_tra none area_variables none missing_extern none missing_resref none ict2_actions none missing_eval none overwriting_file none fun_args warn
-VERSION ~35.15~
+VERSION ~35.15 BWS-FR Fix~
 README ~stratagems/doc/readme-stratagems.html~
 AUTO_EVAL_STRINGS
 
