--- stratagems\spell\use_iwd_chromatic_orb.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\stratagems\\spell\\use_iwd_chromatic_orb.tpa"	
@@ -7,7 +7,7 @@ DEFINE_ACTION_FUNCTION use_iwd_chromatic_orb BEGIN
 	[
 		m_unusable_conjurer=0
 		m.ab_fx.delete{s_opcode=174 OR s_opcode=215}
-		array.new{str_swap}{37800=>14674 37340=>20438 35568=>%stun_string%}
+		array.new{str_swap}{37800=>14674 37340=>20438 35568=>"%stun_string%"}
 		m.ab_fx.alter
 			{
 				PATCH_IF VARIABLE_IS_SET $str_swap("%s_parameter1%") BEGIN 
