--- stratagems\spell\use_iwd_call_lightning.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\stratagems\\spell\\use_iwd_call_lightning.tpa"	
@@ -4,7 +4,7 @@ DEFINE_ACTION_FUNCTION use_iwd_call_lightning BEGIN
 	[
 		m_description:=@2
 		array.new{cl_map}{1=>85 6=>86 7=>87 8=>88 9=>89 10=>90}
-		m.ab.alter{PATCH_IF VARIABLE_IS_SET $cl_map("%p_level%") BEGIN s_projectile=$cl_map("%p_level%") END ELSE BEGIN s_projectile=91 END}
+		m.ab.alter{PATCH_IF VARIABLE_IS_SET $cl_map("%s_level%") BEGIN s_projectile=$cl_map("%s_level%") END ELSE BEGIN s_projectile=91 END}
 	]
 
 END
\ No newline at end of file
