--- stratagems\vampire\ssl\vampire_flight.ssl
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\stratagems\\vampire_flight.ssl"	
@@ -28,7 +28,7 @@ IF TRIGGER
        !GlobalTimerNotExpired("pullback","LOCALS")
        Global("batform","LOCALS",0)
        HPPercentLT(Myself,50)
-       RandomNum(1,2)
+       RandomNum(2,1)
 THEN DO
              Action(Literal)
              SetGlobalTimer("pullback","LOCALS",6)
