--- SubtleD_Stat_Overhauls\SubtleD_Stat_Overhauls.tp2
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\SubtleD_Stat_Overhauls\\SubtleD_Stat_Overhauls.tp2"	
@@ -1,7 +1,7 @@
 BACKUP ~weidu_external/backup/SubtleD_Stat_Overhauls~
 AUTHOR ~SubtleD~
 
-VERSION ~v1.5.4~
+VERSION ~v1.5.4 BWS-FR Fix~
 
 ALWAYS
 
@@ -55,7 +55,11 @@ LANGUAGE
   ~polish~
   ~SubtleD_Stat_Overhauls/lang/en_US/setup.tra~
   ~SubtleD_Stat_Overhauls/lang/pl_PL/setup.tra~
-
+LANGUAGE
+  ~Francais~
+  ~french~
+  ~SubtleD_Stat_Overhauls/lang/en_US/setup.tra~
+  ~SubtleD_Stat_Overhauls/lang/fr_FR/setup.tra~
 
 //__________________________________________________________________________________
 //__________________________________________________________________________________
