--- d5_random_tweaks\comp\random_monster_tweaks.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\d5_random_tweaks\\random_monster_tweaks.tpa"	
@@ -27,7 +27,7 @@ END
 // monster stats_________________________________________________________________________
 //
 DEFINE_ACTION_FUNCTION tweak_monster_stats BEGIN
-COPY_EXISTING_REGEXP GLOB ~^.+\.itm$~ ~override~
+COPY_EXISTING_REGEXP GLOB ~^.+\.cre$~ ~override~
   PATCH_IF (SOURCE_SIZE > 0x71) BEGIN
 	READ_LONG 0x28 animation
 	PATCH_IF (animation = 57949) BEGIN 			// 	yeti
