--- SubtleD_Spell_Tweaks\lib\semi_spont\B3Identify.tph
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\SubtleD_Spell_Tweaks\\lib\\semi_spont\\B3Identify.tph"	
@@ -204,7 +204,7 @@ B3Identify_UnidentifiedCount = -1
 ---------------
 
 function B3Identify_RemainingUsesText()
-	return "Remaining Identify Uses: "..B3Identify_UseCount
+	return "%remaining_uses_text%"..B3Identify_UseCount
 end
 
 function B3Identify_IdentifySlot(slotinfo)
@@ -359,6 +359,8 @@ menu
 		LAF B3_CONTINGENCY_REDIRECT_INSTALL END
 
 		COPY ~.../B3Identify-inlined/copy/M_B3IDEN.lua~ ~override~
+			SPRINT remaining_uses_text @1000
+			EVALUATE_BUFFER
 
 		COPY_EXISTING ~UI.MENU~ ~override~
 
