--- SubtleD_Spell_Tweaks\comp\familiars.tpa
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\SubtleD_Spell_Tweaks\\comp\\familiars.tpa"	
@@ -310,10 +310,14 @@ COPY_EXISTING ~spwi123.spl~ ~override~
   READ_LONG 0x50 desc_strref
 IF_EXISTS BUT_ONLY
 
+OUTER_SPRINT tip  @150100
+OUTER_SPRINT title  @150101
+OUTER_SPRINT label  @150102
+
 LAF CREATE_SEQUENCER_MENU 
   INT_VAR 
 	name = %name_strref%
-	tip = RESOLVE_STR_REF (~Choose a familiar from the following options. This choice is permanent.~) 
+	tip = RESOLVE_STR_REF (~%tip%~)
 	desc = %desc_strref%
 	column = 1
   STR_VAR 
@@ -321,8 +325,8 @@ LAF CREATE_SEQUENCER_MENU
 	icon = ~spwi123b~
 	spelltable = ~d5famtbl~ 
 	spelllist = ~d5famls~
-	title = ~Choose a familiar from the following options~
-	label = ~This choice is permanent~
+	title = EVALUATE_BUFFER ~%title%~
+	label = EVALUATE_BUFFER ~%label%~
 END
 
 ACTION_FOR_EACH num IN ~1~ ~2~ ~3~ ~4~ ~5~ ~6~ ~7~ ~8~ BEGIN
