--- A7-MagicStore\A7-MagicStore.tp2
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\A7-MagicStore\\A7-MagicStore.tp2"	
@@ -1,6 +1,6 @@
 BACKUP ~weidu_external/backup/A7-MagicStore~
 SUPPORT ~https://www.gibberlings3.net/forums/topic/32357-magic-store-of-vergadain-visit-any-store-anywhere-and-anytime~
-VERSION ~3.0~
+VERSION ~3.0 BWS-FR Fix~
 
 // Required by Tutu
 ALLOW_MISSING
