--- cdtweaks\setup-cdtweaks.tp2
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BiG World Fixpack\\cdtweaks\\setup-cdtweaks.tp2"	
@@ -12,7 +12,7 @@ ALWAYS
   
 END
 
-VERSION ~v16~
+VERSION ~v16 BWS-FR Fix~
 
 README ~cdtweaks/languages/%LANGUAGE%/readme-cdtweaks.html~ ~cdtweaks/readme-cdtweaks.html~
 
@@ -3850,11 +3850,13 @@ END
 
 // add portrait change opcodes to edwin transform spells
 COPY_EXISTING ~spin662.spl~ ~override~ // return to edwin
+  LPF DELETE_SPELL_EFFECT INT_VAR opcode_to_delete = 107 END
   LPF ADD_SPELL_EFFECT INT_VAR opcode = 107 target = 2 parameter2 = 1 timing = 1 STR_VAR resource = nedwinm END // large portrait
   LPF ADD_SPELL_EFFECT INT_VAR opcode = 107 target = 2 parameter2 = 0 timing = 1 STR_VAR resource = nedwins END // small portrait
 
 // add portrait change opcodes to edwin transform spells
 COPY_EXISTING ~spin916.spl~ ~override~ // change to edwina
+  LPF DELETE_SPELL_EFFECT INT_VAR opcode_to_delete = 107 END
   LPF ADD_SPELL_EFFECT INT_VAR opcode = 107 target = 2 parameter2 = 1 timing = 1 STR_VAR resource = royo4l END // large portrait
   LPF ADD_SPELL_EFFECT INT_VAR opcode = 107 target = 2 parameter2 = 0 timing = 1 STR_VAR resource = royo4m END // small portrait
 
