--- cdtweaks\dlg\stronghold_dearnise.d
+++ "b/E:\\Modding\\Baldur's Gate 2\\BWS-FR-Fixpack\\BWS-FR-Fixpack\\cdtweaks\\dlg\\stronghold_dearnise.d"	
@@ -1,6 +1,8 @@
 REPLACE_TRIGGER_TEXT ~nalia~    ~Class(Player1,FIGHTER\(_[A-Z]+\)?)~ ~True()~
+REPLACE_TRIGGER_TEXT ~nalia~    ~Kit(Player1,Blackguard)~ ~True()~ // bg2ee
 
 REPLACE_TRIGGER_TEXT ~naliaj~   ~Class(Player1,FIGHTER\(_[A-Z]+\)*)~ ~True()~
+REPLACE_TRIGGER_TEXT ~naliaj~   ~Kit(Player1,Blackguard)~ ~True()~ // bg2ee
 
 ADD_TRANS_ACTION demson 
 BEGIN 119 END
